src/main/java/slimeknights/tconstruct/plugin/crt/managers/MaterialManager.java

```diff
@@ -0,0 +1,40 @@
+package slimeknights.tconstruct.plugin.crt.managers;
+
+import com.blamejared.crafttweaker.api.CraftTweakerAPI;
+import com.blamejared.crafttweaker.api.annotations.ZenRegister;
+import com.blamejared.crafttweaker.api.item.IIngredient;
+import com.blamejared.crafttweaker.api.item.IItemStack;
+import com.blamejared.crafttweaker.api.managers.IRecipeManager;
+import com.blamejared.crafttweaker.impl.actions.recipes.ActionAddRecipe;
+import net.minecraft.item.crafting.IRecipeType;
+import net.minecraft.util.ResourceLocation;
+import org.openzen.zencode.java.ZenCodeType;
+import slimeknights.tconstruct.library.materials.MaterialId;
+import slimeknights.tconstruct.library.recipe.RecipeTypes;
+import slimeknights.tconstruct.library.recipe.material.MaterialRecipe;
+import slimeknights.tconstruct.plugin.crt.CRTHelper;
+
+@ZenRegister
+@ZenCodeType.Name("mods.tconstruct.Material")
+public class MaterialManager implements IRecipeManager {
+  
+  @ZenCodeType.Method
+  public void addMaterial(String name, IIngredient ingredient, int value, int needed, String materialId) {
```

Comment:
With the above rename, you can call this `addItem`. I am looking into making a version of `MaterialRecipe` for fluids.